### PR TITLE
chore: disable auto-close staled issue

### PR DESCRIPTION
**Pull request by UncleBill**
We shouldn't brush the problem under carpet

### CODE REVIEW

**Comment by septs** [comment]
why?

**Comment by UncleBill** [comment]
> why?

Some high priority issues got close automatically. Important problem should not be ignored, we shouldn't pretend they don't exist.

https://github.com/DimensionDev/Maskbook/issues?q=is%3Aissue+sort%3Aupdated-desc+label%3A%22Will+Not+Fix%22+is%3Aclosed+label%3A%22Priority%3A+P0+%28Emergency%29%22

https://github.com/DimensionDev/Maskbook/issues?q=is%3Aissue+sort%3Aupdated-desc+label%3A%22Will+Not+Fix%22+is%3Aclosed+label%3A%22Priority%3A+P1+%28Now%29%22

**Comment by septs** [comment]
https://github.com/DimensionDev/Maskbook/blob/969f506d70ff5182f36d22db711fb18f3ff1d236/.github/stale.yml#L6-L15

plz add `Priority: P0 (Emergency)` and `Priority: P1 (Now)` to `exemptLabels ` list.

File: .github/stale.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # https://github.com/probot/stale
 
 daysUntilStale: 60
-daysUntilClose: 7
+daysUntilClose: false
 
 exemptLabels:
```
Comment (septs) [inline .github/stale.yml:6]: plz update `exemptLabels`!

**Review by septs — CHANGES_REQUESTED** [review]

**Pull request closed without merging**